src/app/wizard/step/provider-settings/provider/extended/openstack/default/component.ts

```diff
@@ -173,6 +185,8 @@ export class OpenstackProviderExtendedDefaultCredentialsComponent
         return this._hasRequiredCredentials() ? '' : 'Please enter your credentials first.';
       case Controls.SubnetID:
         return this._canLoadSubnet() ? '' : 'Please enter your credentials and network first.';
+      default:
+        return null;
```

Comment:
This method should return string, change the signature or this line to return empty string.

Reply:
Is this resolved?